src/prebuild.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs-extra');
+const path = require('path');
+const builder = require('broccoli-builder');
+const rollUpTree = require('./rollup-tree');
+
+module.exports.preBuild = function(modules, preBuildPath, addonPath) {
+    let treeForAddonPromise, treeForVendorPromise;
+    fs.removeSync(preBuildPath);
+
+    const dependencies = rollUpTree.classifyDependencies(modules);
+    if(dependencies.namespacedDependencies.length > 0) {
```

Comment:
as node is only single threaded, I think we should build these two sequentially, not in parallel and prebuild should return a single promise for when prebuild is done.